packages/template-retail-react-app/translations/en-GB.json

```diff
@@ -552,8 +555,8 @@
   "header.button.assistive_msg.my_account": {
     "defaultMessage": "My account"
   },
-  "header.button.assistive_msg.my_cart": {
-    "defaultMessage": "My cart"
+  "header.button.assistive_msg.my_cart_with_num_items": {
+    "defaultMessage": "My cart, number of items: {numItems}"
```

Comment:
Added translations for `en-GB` to clear `Missing Translation` warnings in console when developing